db/data-source.ts

```diff
@@ -0,0 +1,16 @@
+import { DataSource, DataSourceOptions } from "typeorm";
+
+export const dataSourceOptions: DataSourceOptions = {
+    type: 'mysql',
+    host: 'localhost',
```

Comment:
- [ ] Replace host, port, username, password and database values with environment placeholders